theatre/studio/src/panels/SequenceEditorPanel/DopeSheet/Right/BasicKeyframedTrack/KeyframeEditor/CurveEditorPopover/shared.ts

```diff
@@ -0,0 +1,99 @@
+import {useMemo, useRef, useState} from 'react'
+
+// n mod m
+export function mod(n: number, m: number) {
+  return ((n % m) + m) % m
+}
```

Comment:
Suggestion: This is only used in one file, I would strongly recommend keeping it in that one file, so it isn't accidentally imported and reused by another part of the codebase that could misunderstand it's purpose.

Suggestion: If applicable, document or paste link to reference of what this is useful for in regards to the context you want to use it for.